mifosng-android/src/main/java/com/mifos/mifosxdroid/online/CreateNewClientFragment.java

```diff
@@ -438,123 +376,63 @@ public void onNothingSelected(AdapterView<?> parent) {
+                                          officeAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+                                          sp_offices.setAdapter(officeAdapter);
+                                          sp_offices.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+
+                                              @Override
+                                              public void onItemSelected(AdapterView<?> adapterView, View view, int i, long l) {
+                                                  officeId = officeNameIdHashMap.get(officeList.get(i));
+                                                  Log.d("officeId " + officeList.get(i), String.valueOf(officeId));
+                                              }
+
+                                              @Override
+                                              public void onNothingSelected(AdapterView<?> adapterView) {
+
+                                              }
+                                          });
+                                          hideProgress();
+                                      }
+
+                                      @Override
+                                      public void failure(RetrofitError error) {
+                                          hideProgress();
+                                      }
+                                  }
+        );
     }
-    public void inflateStaffSpinner(final int officeId) {
-
-        ((MifosApplication) getActivity().getApplicationContext()).api.staffService.getStaffForOffice(officeId, new Callback<List<Staff>>() {
-            @Override
-            public void success(List<Staff> staffs, Response response) {
 
-                final List<String> staffNames = new ArrayList<String>();
-                for (Staff staff : staffs) {
-                    staffNames.add(staff.getDisplayName());
-                    staffNameIdHashMap.put(staff.getDisplayName(), staff.getId());
-                }
-                ArrayAdapter<String> staffAdapter = new ArrayAdapter<String>(getActivity(),
-                        android.R.layout.simple_spinner_item, staffNames);
-                staffAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
-                sp_staff.setAdapter(staffAdapter);
-                sp_staff.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
-                    @Override
-                    public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
-
-                        staffId = staffNameIdHashMap.get(staffNames.get(position));
-                        Log.d("staffId " + staffNames.get(position), String.valueOf(staffId));
-                        if (staffId != -1) {
-
-                        } else {
-                            Toast.makeText(getActivity(), getString(R.string.error_select_staff), Toast.LENGTH_SHORT).show();
```

Comment:
Hi @ishan1604 noticed that the removal of this staff population method  broke the client creation process , ill fix it when i update the pull request